GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -107,8 +120,14 @@ private void folderGoUpbutton_Click(object sender, EventArgs e)
 
         private void _NO_TRANSLATE_Directory_TextChanged(object sender, EventArgs e)
         {
-            DirectoryInfo currentDirectory = new DirectoryInfo(_NO_TRANSLATE_Directory.Text);
-            folderGoUpbutton.Enabled = (currentDirectory.Parent != null);
+            try
+            {
+                DirectoryInfo currentDirectory = new DirectoryInfo(_NO_TRANSLATE_Directory.Text);
+                folderGoUpbutton.Enabled = (currentDirectory.Parent != null);
```

Comment:
Here I've only moved the existing code into the try/catch because of System.ArgumentException that is raised by DirectoryInfo when `_NO_TRANSLATE_Directory.Text` is empty.
Look at the new commit.

Thank you.